src/main/kotlin/uk/gov/dluhc/notificationsapi/rest/StatisticsController.kt

```diff
@@ -71,10 +74,13 @@ class StatisticsController(
 
         val bespokeCommunications = notifications.filter { it.type == NotificationType.BESPOKE_COMM }.size
 
+        val hasSentInviteToRegister = notifications.any { it.type == NotificationType.INVITE_TO_REGISTER }
+
         return CommunicationsStatisticsResponseOAVA(
```

Comment:
Assume you'll have worked out how to handle this when making the bespokeComms changes before, but just making sure the service APIs will be ready to receive the new shape of CommunicationsStatisticsResponseOAVA/CommunicationsStatisticsResponseVAC without error?

Reply:
Yep I'm working on the service API's now

Reply:
Can merge this in after those if need be

Reply:
yeah I don't think we ever send the CommunicationsStatisticsResponse objects from the service APIs to this one, so probably safest to merge in the service apis first once they're done